gnosis_scan_api.py

```diff
@@ -0,0 +1,81 @@
+import requests
+import json
+import datetime
+import time
+from constants import header, REQUEST_TIMEOUT, SUCCESS_CODE
+
+
+def get_block_range(year, month, day):
+    date_time = datetime.datetime(year, month, day, 0, 0)
+    start_timestamp = int(time.mktime(date_time.timetuple())) + 3 * 60 * 60
+    url = (
+        "https://api.gnosisscan.io/api?module=block&action=getblocknobytime"
+        + "&timestamp="
+        + str(start_timestamp)
+        + "&closest=after&apikey=YourApiKeyToken"
+    )
+    try:
+        response = requests.get(
+            url,
+            headers=header,
+            timeout=REQUEST_TIMEOUT,
+        )
+        if response.status_code == SUCCESS_CODE:
+            data = json.loads(response.text)
+            start_block = int(data["result"])
+    except Exception as e:
+        print(e)
+        return -1, -1
```

Comment:
Are those return values used anywhere? If not, it might be better to let the script crash with a reasonable error message.

```suggestion
        print(f"Fetching block range failed with error: {e})
        raise e
```

Reply:
Yeah makes sense. Pushed a change that also handles the case of future timestamps